### PR TITLE
Adding commuter rail ridership to dashboard

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

Subset of https://github.com/transitmatters/t-performance-dash/pull/898 that adds ridership to the dashboard

## Changes

Screenshot 2024-07-18 at 10 19 52 PM

- Adds Ridership data for commuter rail to data dashboard

## Testing Instructions

Test in beta today

### CODE REVIEW

File: common/components/nav/MenuDropdown.tsx [hunk 1]
```diff
@@ -38,15 +38,20 @@ export const MenuDropdown: React.FC<MenuDropdownProps> = ({ line, route, childre
...
+  const href = React.useMemo(() => {
+    switch (line) {
+      case 'line-bus':
+        return `/bus/trips/single?busRoute=1&date=${BUS_DEFAULTS.singleTripConfig.date}`;
+      case 'line-commuter-rail':
+        return `/commuter-rail/ridership?crRoute=CR-Lowell&startDate=${COMMUTER_RAIL_DEFAULTS.lineConfig.startDate}&endDate=${COMMUTER_RAIL_DEFAULTS.lineConfig.endDate}`;
+      default:
+        return getLineSelectionItemHref(line, route);
+    }
+  }, [line, route]);
```
Comment (skaplan-dev) [inline common/components/nav/MenuDropdown.tsx:50]: could we `util` this?

File: common/components/notices/BetaDataNotice.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const BetaDataNotice: React.FC = () => {
   const isEndDateAfterBusMaxDay = endDate !== undefined && dayjs(endDate).isAfter(BUS_MAX_DAY);
 
   if (
-    (line === 'line-bus' || linePath === 'bus') &&
+    (line === 'line-commuter-rail' || linePath === 'commuter-rail') &&
```
Comment (skaplan-dev) [inline common/components/notices/BetaDataNotice.tsx:23]: do we have these as `consts` somewhere?

File: common/components/notices/GobbleDataNotice.tsx [hunk 3]
```diff
@@ -21,8 +21,10 @@ export const GobbleDataNotice: React.FC = () => {
   const isEndDateAfterBusMaxDay = endDate !== undefined && dayjs(endDate).isAfter(BUS_MAX_DAY);
 
   if (
-    (line === 'line-bus' || linePath === 'bus') &&
-    (isStartDateAfterBusMaxDay || isEndDateAfterBusMaxDay)
+    ((line === 'line-bus' || linePath === 'bus') &&
```
Comment (skaplan-dev) [inline common/components/notices/GobbleDataNotice.tsx:24]: more `const`

**Review by skaplan-dev — APPROVED** [review]
nits

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**